src/__tests__/changeStatus.test.js

```diff
@@ -0,0 +1,35 @@
+import changeStatus from '../__mocks__/changeStatus';
+import localStorageMock from '../__mocks__/localStorage';
+
+describe('test changeStatus function', () => {
+  const arr = [
+    {
+      description: 'something to do 1',
+      completed: true,
+      index: 1,
+    },
+    {
+      description: 'something to do 2',
+      completed: false,
+      index: 2,
+    },
+    {
+      description: 'something to do 3',
+      completed: true,
+      index: 3,
```

Comment:
- [ ] The mock testing should be provided by jest [see here how](https://stackabuse.com/using-mocks-for-testing-in-javascript-with-jes/)

Reply:
Hi, the addTask.test.js run well on my computer. I can no figure what's wrong with our code. How should we do
<img width="555" alt="Screen Shot 2021-09-30 at 3 39 40 PM" src="https://user-images.githubusercontent.com/88653938/135476975-baa1084c-1447-4dfd-91ec-a6f1fe42ad23.png">
?